Sources/BarcodeHeroUI/Views/BHCrosshairView.swift

```diff
@@ -0,0 +1,107 @@
+// Copyright © 2019 SpotHero, Inc. All rights reserved.
+
+#if canImport(UIKit)
+
+    import Foundation
+    import UIKit
+
+    @available(iOS 9.0, *)
+    public class BHCrosshairView: UIView {
+        // MARK: Constants
+
+        private static let cutoutHeight: CGFloat = 170
+        private static let labelHeight: CGFloat = 40
+        private static let width: CGFloat = 255
+
+        // MARK: Properties - Views
+
+        private lazy var barcodeDataLabel: UILabel = {
+            let dataLabel = UILabel()
+            dataLabel.textAlignment = .center
+            dataLabel.translatesAutoresizingMaskIntoConstraints = false
+            dataLabel.heightAnchor.constraint(equalToConstant: Self.labelHeight).isActive = true
+
+            return dataLabel
+        }()
+
+        private lazy var barcodeTypeLabel: UILabel = {
+            let typeLabel = UILabel()
+            typeLabel.textAlignment = .center
+            typeLabel.translatesAutoresizingMaskIntoConstraints = false
+            typeLabel.heightAnchor.constraint(equalToConstant: Self.labelHeight).isActive = true
+
+            return typeLabel
+        }()
+
+        public private(set) lazy var cutoutView: UIView = {
+            let cutoutView = UIView()
+            cutoutView.translatesAutoresizingMaskIntoConstraints = false
+            cutoutView.heightAnchor.constraint(equalToConstant: Self.cutoutHeight).isActive = true
+            cutoutView.widthAnchor.constraint(equalToConstant: Self.width).isActive = true
+
+            return cutoutView
+        }()
+
+        // MARK: Properties - Convenience
+
+        public var barcodeData: String? {
+            get {
+                return self.barcodeDataLabel.text
+            }
+            set {
+                self.barcodeDataLabel.text = newValue
+            }
+        }
+
+        public var barcodeType: String? {
+            get {
+                return self.barcodeTypeLabel.text
+            }
+            set {
+                self.barcodeTypeLabel.text = newValue
+            }
+        }
+
+        // MARK: Methods - Lifecycle
+
+        override public func layoutSubviews() {
```

Comment:
This seems like initialization code that shouldn't be in a function like `layoutSubviews`?

Reply:
Great catch, not sure why it's in here.